features/networking/multus.feature

```diff
@@ -991,3 +991,36 @@ Feature: Multus-CNI related scenarios
     When I execute on the pod:
       | /usr/sbin/ip | a |
     Then the output should contain "192.168.1"
+
+  # @author weliang@redhat.com
+  # @case_id OCP-25910
+  @admin
+  Scenario: Assign static MAC address using pod annotation				 
+    # Make sure that the multus is enabled
+    Given the multus is enabled on the cluster
+
+    # Create the net-attach-def via cluster admin
```

Comment:
@weliang1....I think you forgot to insert the line `Given I switch to cluster admin pseudo user`

Reply:
Do not need to, line 1004 "When I run the :create admin command with" will make it work